files/wingedhelix_cluster/ldap.crt

```diff
@@ -0,0 +1,105 @@
+$ANSIBLE_VAULT;1.2;AES256;wingedhelix_cluster
```

Comment:
You are correct. The *.crt can be public, but to prevent other stuff related to the certs not being encrypted when it should be encrypted, we currently simply encrypting everything related to certs, corresponding key pairs, etc. There are also container file formats to store certificate chains and where multiple cert plus private keys can be stored in a single file. It can be confusing ...